library/src/morpheus-master/sass/modules/_footer.scss

```diff
@@ -36,6 +36,11 @@
 	}
 	
 	##{$namespace}footer--poweredBy {
```

Comment:
Too many # signs.

Reply:
That element is an ID in the dom so I think this is correct.

Reply:
Oh, my bad!